include/procfile.bash

```diff
@@ -18,7 +18,7 @@ procfile-exec() {
 	procfile-load-env
 	procfile-load-profile
 	cd "$app_path"
-	unprivileged /bin/bash -c "$(eval echo $@)"
+	exec setuidgid "$unprivileged_user" $(eval echo $@)
```

Comment:
So the main thing here is that you're not running Bash. In that case, there shouldn't be a problem using `unprivileged`. I first thought this would break compatibility because Procfiles let you include Bash so it needs to be run under Bash, but it turns out `eval` takes care of that. `eval` was never ideal, but if I recall it's the only thing that did the trick.